llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -998,14 +995,29 @@ static bool isGlobalVarSummary(const ModuleSummaryIndex &Index,
   return false;
 }
 
-template <class T>
-static unsigned numGlobalVarSummaries(const ModuleSummaryIndex &Index, T &Cont,
+static unsigned numGlobalVarSummaries(const ModuleSummaryIndex &Index,
```

Comment:
I think this can be simplified more - simply return the number that are GVs (i.e. no output reference parameters needed). The number that are functions can be deduced by the caller from the size of the ExportSet minus the returned GVs count.